README.md

```
@@ -29,22 +61,81 @@ all classes  |	100% (6/ 6)  |	93.6% (44/ 47) |  96.2% (332/ 345)  |
+    <groupId>com.fasterxml.jackson.core</groupId>
+    <artifactId>jackson-databind</artifactId>
+    <version>2.18.2</version>
+</dependency>
+<dependency>
+    <groupId>tools.jackson.core</groupId>
+    <artifactId>jackson-databind</artifactId>
+    <version>3.0.0</version>
+</dependency>
+```
+
 - 0.4.16 and below can be found at https://central.sonatype.com/artifact/com.flipkart.zjsonpatch/zjsonpatch/overview
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)